### ISSUE TITLE
BullsEye: font size within bubbles too small

### ISSUE THREAD

**Issue by hschirmacher** [chmac/cohesion-catalyst#10]
On the BullsEye page (on :3000/bullseye, user "bullseye", pw "password") on some 4K monitors the font size of the labels within the circles is way too small, so that the text is not readable. Tried changing the desktop resolution under Ubuntu as well as the zoom level of the Chrome browser without a satisfactory result. 

Then tried to "hack" a *3 into the code into the function that determines the font size in app/client/templates/application/bullseye/bullseye_bubblebath.js . This seemed to work fine. 

Suggestion: add an input field for font scale factor N on the "bulls eye" admin page, store N in the admin database, and add "*N" into the above mentioned code, so that one can adapt the font size via the admin interface.

**Comment by chmac** [comment]
This looks quite tricky. There don't seem to be any settings in this part of the code. I'm also not 100% sure where exactly you put in the `* 3`. I thought there would be an easy way to add this as an option, but it doesn't look like it.

We could do something simple like have a flag in the code which we can set on or off. But that's really just a slightly easier to modify version of what you already did.

I'm not sure how the settings which are on the admin view are loaded on the client. I'd need to figure all that out, plus somehow load the variable when the bullseye template is loading, and probably delay the whole rendering until this value has loaded. That's not super obvious. I don't believe the d3 code that runs where the `* 3` is in effect is a "reactive context" in meteor parlance.

Bottom line, I don't see an easy, quick fix approach that will work here unfortunately.
